guides/leave_space_mutation.md

```diff
@@ -0,0 +1,50 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'leave-space-mutation'
+is: 'experimental'
+---
+
+# Leave Space
+
+## Concepts
+
+The _leaveSpace_ mutation allows you to leave a workspace, where applicable, with the Space ID. Currently this only applicable to spaces you are currently invited to.
+In this scenario _leaveSpace_ is used to reject the invite. 
```

Comment:
what does this sentence mean - isn't this ALWAYS how one uses the _leaveSpace_ mutation? If so just delete this sentence.

Reply:
If you have an invite for a space leaveSpace will be used to reject the invite. For anyone else calling the API they will get an error message. We have plans to extend the API to all space members so anyone can call leaveSpace to leave the space instead of updating the space member list like they do today. I can remove the sentence for now if it's confusing.

Reply:
How about something more like

> Currently this API only supports invited space members, for other members it will return an error message. The _leaveSpace_ mutation allows you to leave a workspace. The mutation accepts a Space ID as an argument and makes the request on behalf of the calling user.

Reply:
Ok, that works

Reply:
For consistency with `joinSpace` above how about:
"The _leaveSpace_ mutation allows the user to leave a space. Currently this API can only be used to reject an invitation to a space."